model/resolver/Resolver.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class Resolver
+{
+
+    private function startsWith(string $prefix, string $target) : bool {
+        return strpos($target, $prefix) === 0;
+    }
+
+    /**
+     * Resolve the URI using the most appropriate resolver and return the
+     * result as an ExternalResource.
+     * @param string $uri URI to resolve
+     * @return EasyRdf\Resource
+     */
+    public function resolve(string $uri, int $timeout): ?EasyRdf\Resource {
+        if (preg_match('|http://id.loc.gov/[^/]+/[^/]+/.+|', $uri)) {
+            $res = new LOCResource($uri);
+        } elseif ($this->startsWith('http://www.wikidata.org/entity/', $uri)) {
+            $res = new LDFResource($uri, 'https://query.wikidata.org/bigdata/ldf');
+        } else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method resolve uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/manual/NatLibFi/Skosmos/pullRequest?prid=4951447)